src/redux/actions/user-actions.js

```diff
@@ -7,3 +7,11 @@ export const fetchUser = () => ({
     resolve(UserHelper.fetchUser());
   })
 });
+
+export const fetchUsers = () => ({
+  type: ActionTypes.FETCH_USERS,
+  payload: new Promise(resolve => {
```

Comment:
Isn't `payload: UserHelper.fetchUsers()` enough? If not can you please add `await` and `async` to `fetchUsers` function.

Reply:
Yep, `payload: UserHelper.fetchUsers()` is enough